mcos/error_estimator/variance.py

```diff
@@ -0,0 +1,10 @@
+import numpy as np
+
+from mcos.error_estimator import AbstractErrorEstimator
+
+
+class VarianceErrorEstimator(AbstractErrorEstimator):
+    """Error Estimator that calculates the mean difference in variance"""
+
+    def estimate(self, mu: np.array, cov: np.array, allocation: np.array, optimal_allocation: np.array) -> float:
+        return np.dot(optimal_allocation - allocation, np.dot(optimal_allocation - allocation, cov))
```

Comment:
This calculates the
<img width="486" alt="Screen Shot 2020-01-29 at 2 11 28 PM" src="https://user-images.githubusercontent.com/24689747/73388990-3f74ab80-42a1-11ea-8aa9-623199277444.png">
where the first `w` is the optimal allocation, the second `w` is the allocation and the `V` is the covariance matrix.